hoot-core/src/main/cpp/hoot/core/elements/ElementDeduplicator.cpp

```diff
@@ -107,12 +107,12 @@ void ElementDeduplicator::dedupe(OsmMapPtr map1, OsmMapPtr map2)
   LOG_DEBUG(map2->getName() << " size before de-duping: " << map2->size());
   LOG_VARD(_favorMoreConnectedWays);
 
-  const long map1NodesBefore = map1->getNodeCount();
-  const long map1WaysBefore = map1->getWayCount();
-  const long map1RelationsBefore = map1->getRelationCount();
-  const long map2NodesBefore = map2->getNodeCount();
-  const long map2WaysBefore = map2->getWayCount();
-  const long map2RelationsBefore = map2->getRelationCount();
+  const int map1NodesBefore = map1->getNodeCount();
+  const int map1WaysBefore = map1->getWayCount();
+  const int map1RelationsBefore = map1->getRelationCount();
+  const int map2NodesBefore = map2->getNodeCount();
+  const int map2WaysBefore = map2->getWayCount();
+  const int map2RelationsBefore = map2->getRelationCount();
```

Comment:
All of these are like the comment above, keep them `long` and then cast after they are calculated and saved.